helpers/deploymentConfig.ts

```diff
@@ -407,130 +447,154 @@ export const bridgeConfig: BridgeConfig = {
+      ...createMethodEntries(parseInt(preconfiguredAddresses.sepolia.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.arbitrumsepolia.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.zksyncsepolia.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.opsepolia.LzVirtualChainId)),
     ],
   },
   basemainnet: {
     methods: [
-      ...createMethodEntries(102),
-      ...createMethodEntries(101),
-      ...createMethodEntries(202),
-      ...createMethodEntries(165),
-      ...createMethodEntries(111),
-      ...createMethodEntries(110),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.bscmainnet.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.ethereum.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.opbnbmainnet.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.arbitrumone.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.zksyncmainnet.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.opmainnet.LzVirtualChainId)),
+      ...createMethodEntries(
+        parseInt(preconfiguredAddresses.unichainmainnet.LzVirtualChainId),
+        300000,
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+      ),
     ],
   },
   unichainsepolia: {
     methods: [
       ...createMethodEntries(
-        10102,
+        parseInt(preconfiguredAddresses.bsctestnet.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        10202,
+        parseInt(preconfiguredAddresses.opbnbtestnet.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        10161,
+        parseInt(preconfiguredAddresses.sepolia.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        10231,
+        parseInt(preconfiguredAddresses.arbitrumsepolia.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        10248,
+        parseInt(preconfiguredAddresses.zksyncsepolia.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
```

Comment:
I miss the entry for `basesepolia`

Reply:
https://github.com/VenusProtocol/token-bridge/pull/49/commits/6d39c6357cec3cb1b89a9417aadc458875b50841